src/ledger/SharedModuleCacheCompiler.h

```diff
@@ -17,9 +17,13 @@
 #include <mutex>
 #include <thread>
 
+namespace medida
+{
+class MetricsRegistry;
```

Comment:
nit: Does this really need to be forward-declared here?

Reply:
Fixed